jerry-core/lit/lit-char-helpers.c

```diff
@@ -481,6 +685,55 @@ lit_char_to_lower_case (ecma_char_t character, /**< input character value */
     return 1;
   }
 
+#ifndef CONFIG_DISABLE_UNICODE_CASE_CONVERSION
+
+  ecma_length_t lowercase_sequence = search_in_character_case_ranges (character, output_buffer_p, true);
+
+  if (lowercase_sequence != 0)
+  {
+    return lowercase_sequence;
+  }
+
+  lowercase_sequence = search_in_character_pair_ranges (character, output_buffer_p, true);
+
+  if (lowercase_sequence != 0)
+  {
+    return lowercase_sequence;
+  }
+
+  lowercase_sequence = search_in_character_pairs (character, output_buffer_p, true);
+
+  if (lowercase_sequence != 0)
+  {
+    return lowercase_sequence;
+  }
```

Comment:
It seems to me that the 3 function call is the same as the uppercase variant, so could we combine those 3 functions together?